cookbooks/cdo-varnish/templates/default/default.vcl.erb

```diff
@@ -98,7 +98,15 @@ sub vcl_recv
   else if(req.http.Host == "<%= domain %>")
   {
     /* <%= domain %> redirects to <%= site %> */
-    return(synth(751, "https://<%= site %>" + req.url));
+    if("<%= site %>" ~ "^http://")
```

Comment:
Since is filtering the ERB variable rather than VCL, I'd prefer this filter be done directly the ERB template logic (in Ruby within a `<% ... %>` block) rather than pushed to the VCL output.

Something like:

```erb
<%
  redirects.each_pair do |domain, site|
    site = "https://#{site}" unless site =~ %r[^https?://]
%>
  return(synth(751, "<%= site %>" + req.url));
<% end %>
```

Reply:
Ah, I wondered about that.  Good call.